storage/log_storage.go

```diff
@@ -136,4 +136,8 @@ type LogRootWriter interface {
 type LogMetadata interface {
 	// GetActiveLogs returns a list of the IDs of all the logs that are configured in storage
 	GetActiveLogIDs(ctx context.Context) ([]int64, error)
+
+	// GetUnsequencedCounts returns a map of the number of unsequenced entries
+	// by log ID.
+	GetUnsequencedCounts(ctx context.Context) (map[int64]int64, error)
```

Comment:
The particular use-case for this is to background export this map of logID->count as monitoring metrics, so not particularly useful to pull them out individually at the moment...